stdlib/ssl.pyi

```diff
@@ -362,6 +400,12 @@ if sys.version_info >= (3, 5):
         if sys.version_info >= (3, 6):
             session: Optional[SSLSession]
             session_reused: bool
+        if sys.version_info >= (3, 7):
+            def __init__(self, *args: Any, **kwargs: Any) -> None: ...
+        else:
```

Comment:
As `__init__()` can't be called in 3.7, I think is better to you leave it out (like you did with `SSLSocket`):

```suggestion
        # __init__() can't be called starting with Python 3.7
        if sys.version_info < (3, 7):
```

Reply:
We don't have a way to say "this class cannot be instantiated" in stubs, do we? Perhaps putting a NoReturn return type on `__new__` would do it.

Reply:
I don't think we do. I doubt `NoReturn` would work on all type checkers, although it's a good idea. Maybe something to bring up on typing-sig?

Reply:
Well, considering we currently have no way to model a raising constructor as even mypy gives an error:
main.py:17: error: The return type of "__init__" must be None

I disagree with srittau suggestion since you'll get a default constructor anyway - at least this one matches with the definition.

Reply:
In that case, we should add a constructor to `SSLSocket` as well, for consistency's sake.